criteria/open-to-contributions.md

```diff
@@ -6,14 +6,14 @@ order: 4
 
 ## Requirements
 
-
-* The codebase MUST have a public issue tracker that allows suggestions from anyone
+* The codebase MUST have a public issue tracker that accepts suggestions from anyone
 * The codebase MUST allow anyone to submit suggestions for changes to the codebase
-* The documentation MUST link to both the public issue tracker and submitted codebase changes
+* The documentation MUST link to both the public issue tracker and submitted codebase changes, for example in a `README` file
 * The codebase MUST include an email address for security issues and responsible disclosure
-* The codebase MUST include contribution guidelines explaining how contributors can get involved
+* The codebase MUST include contribution guidelines explaining how contributors can get involved, for example in a `CONTRIBUTING` file
 * The codebase SHOULD have a publicly available roadmap
-* The codebase SHOULD advertize the committed engagement of involved organizations in the development and maintenance
+* The codebase SHOULD advertise the committed engagement of involved organizations in the development and maintenance
+* The documentation should include an explanation of how security disclosures will be handled
```

Comment:
Explaining how the reported vulnerabilities are handled is only one aspect of what would make a responsible disclosure process effective. I think there should be additional requirements in this area.

I would suggest requiring a fully fledged policy instead of an explanation. And adding specific requirements of what this policy should at least include.

Reply:
I do like this, however, there might be some cases in which this means that the organization behind the codebase is by law required to prosecute disclosure, even if it is accidental. If this is the case some projects might want to stay a bit vaguer here so that potential contributors are not scared off making issues.

Also, the `SHOULD` here MUST be capitalized ;)

Reply:
In https://github.com/sfosc/sfosc/issues/85#issuecomment-504207684 @lirantal shared responsible disclosure policies he's worked on. Perhaps the language from the policy created for verdaccio works better for this concern?
https://github.com/verdaccio/verdaccio/blob/master/SECURITY.md

It keeps terms vague to "best effort" and "reasonable time" which seem safe enough to adopt. The stronger requirements are delegated to the HackerOne and Snyk security programs.

Reply:
@bvhme well explained, you got it right 👍

Reply:
I've updated the content now based on this discussion. @beanow, in line with your suggestion I've said the project should have a security policy, but I haven't set out what needs to be in it. Instead, I've added a link to Verdacchio's in the Further reading.